app/assets/stylesheets/application.scss

```diff
@@ -10,31 +11,82 @@ $font-stack: "PT Mono";
 // $primary-brand-color
 // $secondary-brand-color
 
+h1, h2, h3, h4, h5 {
+  font-family: 'Lora', serif;
+}
+
 $primary-background-color: $pastel-orange; //  pastel-orange
 $secondary-background-color: $light-green; // light-green
 // in overrides
-// $primary-color: #F35F55; //  pastel-red
-// $secondary-color: #335650; //dark-green
+$primary-color: #F35F55; //  pastel-red
+$secondary-color: #335650; //dark-green
 
 // Styles
 body {
-
+  background-color: #eaeee8;
   font-family: $font-stack;
   font-weight: normal;
 }
 
+a:hover, a:focus {
+    color: #FFFFFF;
```

Comment:
Color literals like `#FFFFFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFFFFF` should be written as `#ffffff`<br>Line should be indented 2 spaces, but was indented 4 spaces